src/utils/countInputValidation.js

```diff
@@ -0,0 +1,12 @@
+import { ERROR_MESSAGES } from "../constants/messages.js";
+
+const countInputValidation = (COUNT) => {
+  if (isNaN(COUNT)) {
+    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.ONLY_NUMBER);
+  }
+  if (Number(COUNT) < 1) {
+    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.NOT_NEGATIVE);
+  }
```

Comment:
```suggestion
  if (Number(COUNT) < 1) {
    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.NOT_NEGATIVE);
  }
```
이 부분도 생각 못했던 것 같은데 좋은 것 같습니다~!